Makefile

```diff
@@ -724,6 +724,15 @@ example-evaluator-binaries: examples/evaluators/golang/simple/simple examples/ev
 # For presubmit we want to update the protobuf generated files and verify that tests are good.
 presubmit: sync-deps clean-protos all-protos fmt vet build test
 
+build/release/: presubmit clean-install-yaml install/yaml/
+	mkdir -p $(BUILD_DIR)/release/
+	cp install/yaml/* $(BUILD_DIR)/release/
+
+release: build/release/
```

Comment:
Should this be setting the tag and the repository?  I have a change out to properly set the repository when building install/yaml/, because currently it points to the main images, which doesn't work when iterating.